nc models/app.model.js

```diff
@@ -63,3 +63,20 @@ exports.insertComment = ({username, body, article_id}) => {
     return rows[0];
   })
 }
+
+exports.updateArticleVotes = ({article_id, inc_votes}) => {
+  if (typeof inc_votes !== "number") {
+    return Promise.reject({status:400, msg:"Invalid input"});
+}
```

Comment:
You don't need to manually check this - if you attempt the query with an invalid data type then psql will throw an error for you itself. Remove the above and see what happens / what you can catch in the controller.